src/apps/miscapps/generateBaseCellNeighbors.c

```diff
@@ -79,6 +80,17 @@ void auditBaseCellNeighbors(int baseCellNeighbors[NUM_BASE_CELLS][7],
     }
 }
 
+char* getBaseCellOutput(baseCell) {
```

Comment:
Please add an explicit type for `baseCell`

Reply:
Oh wow, I didn't even notice that and I'm amazed that compiled.